sormas-api/src/main/java/de/symeda/sormas/api/utils/DataHelper.java

```diff
@@ -71,8 +71,8 @@ public static String createUuid() {
 		return uuid;
 	}
 
-	public static String createConstantUuid(int seed) {
-		return new UUID(0, seed).toString();
+	public static String createConstantUuid() {
```

Comment:
I think we should leave the possibility of having different constant uuids, otherwise this method will always result in `00000000-0000-0000-0000-000000000000` which in the end could just return this hardcoded string.

We should still keep different constant uuids for the default data, so for example it's ok to always have the default region uuid `00000000-0000-0000-0000-000000000001` while the default district uuid is `00000000-0000-0000-0000-000000000002` for example.

Reply:
done